rfc/text/advanced/ap_10_rpc_progressive_call_invocations.md

```diff
@@ -282,6 +282,25 @@ The sequence diagram below illustrates this sitation, where the Network actor mo
      `------'     `------'        `-------'       `------'
 
 
+**Non-sequential CALL Request IDs**
+
+When sending a CALL that continues a progressive call invocation, its request ID is the same as the CALL that initiated the progressive call invocation. Therefore, when progressive call invocations are enabled, the requirement for CALL request IDs to be sequential session scope is waived (see [Protocol Violations](#protocol_errors)). Only those CALL messages which initiate a new RPC shall be required to have a sequential session scope request ID. A Router MAY choose to remember request IDs of previously completed progressive invocations calls for verification purposes, but this is not required.
```

Comment:
> A Router MAY choose to remember request IDs of previously completed progressive invocations calls for verification purposes, but this is not required.

Router -> Dealer

Also I think it's worth noting that if a dealer wants to verify whether a non-sequential id is from a previously completed progressive call, care must be taken to find the right balance in the time these id's are remembered. Too short and you can have data races with slow networks and wrongly flag invalid call id's. Too long and you might end up with a memory overflow especially when lots of progressive calls are made.

Reply:
> When sending a CALL that continues a progressive call invocation

"invocation" seems redundant here. I think you can leave it out.

Reply:
It appears you reviewed an older version of the text, but I applied your suggestions to the current version in this PR.

Reply:
Yea it looks like I did. I'm still getting used to reviewing using GitHub since I'm used to a different system... Not sure how to review multiple commits of the same pr at the same time yet, but managed to view your latest changes. And to me it looks good now 😊

Reply:
The next time, you can go to the Files Changes tab where it shows the changes for all commits at once. You can then click on a + or - to comment and start a review.